master/reference/cni-plugin/configuration.md

```diff
@@ -377,3 +379,54 @@ You can request a floating IP address for a pod through [Kubernetes annotations]
    > **Warning**: This feature can allow pods to receive traffic which may not have been intended for that pod.
    > Users should make sure the proper admission control is in place to prevent users from selecting arbitrary floating IP addresses.
    {: .alert .alert-danger}
+
+### Using IP pools node selectors
+
+You can configure your cluster so that workloads running a certain node will
```

Comment:
I think this first sentence should be tweaked, since workloads don't run nodes.

Maybe this should be "workloads running on certain nodes"?

Reply:
I'm also wondering if the framing of this is right, since using node selectors isn't an optional feature (like the two options above), but actually a required one, since _all_ IP pools will have a node selector, defaulted to "all()".

So rather than saying "You can configure it so certain workloads will only allocate from a specific IP pool", we might want to frame it as "Nodes will only assign workload addresses from IP pools which select them. By default IP pools select all nodes, but this can be configured using the `nodeSelector` field" - with a link to the IP pool documentation.